.github/workflows/ci.yml

```diff
@@ -35,6 +35,26 @@ jobs:
           command: ${{matrix.command}}
           args: "${{matrix.command == 'fmt' && '--all -- --check' || '-- -D warnings'}}"
 
+  brew:
+    name: brew
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        hdf5: [hdf5@1.8, hdf5@1.10, hdf5@1.12, hdf5-mpi]
+        rust: [stable]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with: {submodules: true}
+      - name: Install Rust (${{matrix.rust}})
+        uses: actions-rs/toolchain@v1
+        with: {toolchain: '${{matrix.rust}}', profile: minimal, override: true}
+      - name: Install HDF5 (${{matrix.hdf5}})
+        run: brew install ${{matrix.hdf5}}
+      - name: Build and test all crates
+        run: cargo test -vv
```

Comment:
This is not actually enabling an MPI build as you have to pass in `--features` (see like how it's done below)